src/ai-did-you-mean-this/azext_ai_did_you_mean_this/data/top_3_recommendations.json

```diff
@@ -0,0 +1,11140 @@
+{
+    "2.1.0": {
+        "ad group owner": {
+            "": [
+                {
+                    "SuccessCommand": "ad group show",
+                    "SuccessCommand_Parameters": "--group",
+                    "SuccessCommand_ArgumentPlaceholders": "MyGroupDisplayName",
+                    "NumberOfPairs": 16431
+                }
+            ]
+        },
```

Comment:
This is file could be used when Aladdin service not available? But not used now?